src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1499,6 +1518,14 @@ public void trackNotification(final String eventName, final InAppNotification no
         }
 
         @Override
+        public void joinExperimentIfAvailable() {
+            final JSONArray variants = mDecideMessages.getVariants();
+            if (null != variants) {
+                mUpdatesFromMixpanel.setVariants(variants);
```

Comment:
Event `setVariants` looks weird to me. We are not loading all the different versions to the app, are we?